apps/ci/ci-script.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echo "compile core"
+export CCACHE_CPP2=true
+ccache -s
+timeout 2580 bash ./acore.sh "compiler" "all"
+ccache -s
+
+echo "start worldserver dry-run"
```

Comment:
this `echo` should be inside the `if` block

Reply:
Fixed this, thanks